src/main/java/pl/javastart/restoffers/offer/OfferService.java

```diff
@@ -0,0 +1,42 @@
+package pl.javastart.restoffers.offer;
+
+import org.springframework.stereotype.Service;
+import pl.javastart.restoffers.category.Category;
+import pl.javastart.restoffers.category.CategoryRepository;
+
+import java.util.Optional;
+
+@Service
+public class OfferService {
+
+    private final OfferRepository offerRepository;
```

Comment:
Jeżeli masz klasę Service, to w kontrolerze powinnaś z niej korzystać. Repozytorium powinno być ukryte